app/coffee/Features/Subscription/SubscriptionController.coffee

```diff
@@ -14,10 +14,7 @@ module.exports = SubscriptionController =
 
 	plansPage: (req, res, next) ->
 		plans = SubscriptionViewModelBuilder.buildViewModel()
-		if AuthenticationController.isUserLoggedIn(req)
-			baseUrl = ""
-		else
-			baseUrl = "/register?redir="
```

Comment:
is this /register not need any more. 

It looks like baseURL isn't used at all if that is the case.